### PR TITLE
Adds Fix All button action

### PR DESCRIPTION
**Pull request by emilylaguna**
Project: #15190
Related PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/329

### To test:
1. Launch the app
2. Tap the My Site tab
3. Tap on a Jetpack Site with fixable threats
4. You should see a 'Fix All' button
5. Tap it
6. You should see a localized confirmation alert
7. Tap Cancel
8. Nothing should happen
9. Tap 'Fix All' Again
10. Tap on the 'Fix all threats' button
11. It will appear as if nothing happened
12. Refresh after a minute and there should be no fixable threats

### PR submission checklist:

- [ ] I have considered adding unit tests where possible.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift [hunk 1]
```diff
@@ -132,4 +172,26 @@ class JetpackScanCoordinator {
     private struct Constants {
         static let refreshTimerInterval: TimeInterval = 5
     }
+
+
+    private struct Strings {
+        static let fixAllAlertTitleFormat = NSLocalizedString("Please confirm you want to fix all %1$d active threats", comment: "Confirmation title presented before fixing all the threats, displays the number of threats to be fixed")
+        static let fixAllSingleAlertTitle = NSLocalizedString("Please confirm you want to fix all %1$d active threat", comment: "Confirmation title presented before fixing a single threat")
```
Comment (momo-ozawa) [inline WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift]: Single threat string 👀 
```suggestion
        static let fixSingleAlertTitle = NSLocalizedString("Please confirm you want to fix this threat", comment: "Confirmation title presented before fixing a single threat")
```

File: WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift [hunk 2]
```diff
@@ -132,4 +172,26 @@ class JetpackScanCoordinator {
...
+        static let fixAllAlertTitleMessage = NSLocalizedString("Jetpack will be fixing all the detected active threats.", comment: "Confirmation message presented before fixing all the threats, displays the number of threats to be fixed")
+        static let fixAllSingleAlertMessage = NSLocalizedString("Jetpack will be fixing all the detected active threat.", comment: "Confirmation message presented before fixing a single threat")
```
Comment (momo-ozawa) [inline WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift]: Single threat string 👀 
```suggestion
        static let fixSingleAlertMessage = NSLocalizedString("Jetpack will be fixing the detected active threat.", comment: "Confirmation message presented before fixing a single threat")
```

**Review by momo-ozawa — COMMENTED** [review]
@emilylaguna 

> Tap on the 'Fix all threats' button
> It will appear as if nothing happened
> Refresh after a minute and there should be no fixable threats

I tested on the Digital Ocean Jetpack test site, and wasn't able to get rid of the fixable threats after refreshing.

Also, I found it a bit confusing how the message says "5 potential threats", but the alert shows "3 active threats". It would be nice to match the web and display "Jetpack can autofix 3 of 5 found threats". Not sure if you were already planning on addressing this in a different PR, but just in case!

Screen Shot 2021-01-19 at 10 45 28

File: WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift (hunk 2 above)
Comment (emilylaguna) [reply WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift]: I'd been staring at strings for too long 😆 Fixed in: f67b8205b767420aec81abc1f6f6baff86188e82

File: WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift (hunk 1 above)
Comment (emilylaguna) [reply WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift]: 🤦‍♀️ Good catch. Fixed in: f67b8205b767420aec81abc1f6f6baff86188e82

**Comment by emilylaguna** [comment]
>I tested on the Digital Ocean Jetpack test site, and wasn't able to get rid of the fixable threats after refreshing.

That's odd, do these threats get fixed when trigger on the web?

**Comment by emilylaguna** [comment]
> Also, I found it a bit confusing how the message says "5 potential threats", but the alert shows "3 active threats". It would be nice to match the web and display "Jetpack can autofix 3 of 5 found threats". Not sure if you were already planning on addressing this in a different PR, but just in case!

The alert confirmation uses the wording from the web and only displays the number of threats that can be fixed. I can address this in a later PR when we do design review.

**Review by leandroalonso — APPROVED** [review]
@momo-ozawa Did a small exploration here and the fix is not working because the SFTP user doesn't have write permission. On the web there's a notice stating that "not all threats were fixed".

@emilylaguna is this part of handling errors and it will be addressed in another PR?

Besides that, tested the instructions on another site and it worked just fine.

**Comment by emilylaguna** [comment]
> @emilylaguna is this part of handling errors and it will be addressed in another PR?

Yep! I have a whole task for error handling for the section.

**Pull request merged**